src/app/assessment/assessment.service.ts

```diff
@@ -0,0 +1,483 @@
+          handle: "1OJvu3EQTZmjDgrHPDzj",
+          key: "case/uploads/public/WsLzxK4R3CX1QYGekIau_sample2.mp4",
+          mimetype: "video/mp4",
+          originalFile: {name: "sample2.mp4", type: "video/mp4", size: 788493},
+          originalPath: "sample2.mp4",
+          size: 788493,
+          source: "local_file_system",
+          status: "Stored",
+          uploadId: "75d47c172b9b7f73073fbadd691f3b4f1",
+          url: "https://cdn.filestackcontent.com/1OJvu3EQTZmjDgrHPDzj"
+        }
+      },
+      6: {
+        answer: {
+          container: "practera-aus",
+          filename: "sample.jpg",
+          handle: "EjdSc3EJSLmJnu4WVJnh",
+          key: "case/uploads/public/4S5U3B8TKWcRpifXA6ym_sample.jpg",
+          mimetype: "image/jpeg",
+          originalFile: {name: "sample.jpg", type: "image/jpeg", size: 51085},
+          originalPath: "sample.jpg",
+          size: 51085,
+          source: "local_file_system",
+          status: "Stored",
+          uploadId: "ca9dc1b2f1ae59d14f242b691f4bc6a7",
+          url: "https://cdn.filestackcontent.com/EjdSc3EJSLmJnu4WVJnh"
+        }
+      },
+      7: {
+        answer: {
+          container: "practera-aus",
+          filename: "sample2.mp4",
+          handle: "1OJvu3EQTZmjDgrHPDzj",
+          key: "case/uploads/public/WsLzxK4R3CX1QYGekIau_sample2.mp4",
+          mimetype: "video/mp4",
+          originalFile: {name: "sample2.mp4", type: "video/mp4", size: 788493},
+          originalPath: "sample2.mp4",
+          size: 788493,
+          source: "local_file_system",
+          status: "Stored",
+          uploadId: "75d47c172b9b7f73073fbadd691f3b4f1",
+          url: "https://cdn.filestackcontent.com/1OJvu3EQTZmjDgrHPDzj"
+        }
+      }
+    }
+  };
+
+  // assessment_id: {
+  //  id:
+  //  answers: {
+  //    question_id: {}
+  //  }
+  // }
+  reviews = {
+    3: {
+      id: 1
+    },
+    4: {
+      id: 1,
+      answers: {
+        1: {
+          answer: 'reviewer answer for question 1',
+          comment: 'reviewer comment for question 1'
+        },
+        3: {
+          answer: 1,
+          comment: 'reviewer comment. You are wrong'
+        },
+        2: {
+          comment: 'it will be better if you answer this question'
+        },
+        4: {
+          answer: [2,3],
+          comment: 'reviewer comment. Not 100% correct'
+        }
+      }
+    },
+    5: {
+      id: 2,
+      answers: {
+        1: {
+          answer: 'reviewer answer for question 1',
+          comment: 'reviewer comment for question 1'
+        },
+        3: {
+          answer: 1,
+          comment: 'reviewer comment. You are wrong'
+        },
+        2: {
+          comment: 'it will be better if you answer this question'
+        },
+        4: {
+          answer: [2,3],
+          comment: 'reviewer comment. Not 100% correct'
+        }
+      }
+    }
+  };
+
+  // review id: 
+  feedbackReviewed = {
+    1: false,
+    2: true
+  }
+
+  constructor() {};
```

Comment:
@shawnm0705 just a note, the semicolon after the closing curly bracket isn't necessary.